### PR TITLE
raidboss: added missing translations Fr on Trial shb

### PR DESCRIPTION
**Pull request by VanSlanzar**

### CODE REVIEW

File: ui/raidboss/data/04-sb/trial/rathalos-ex.ts [hunk 1]
```diff
@@ -69,6 +70,7 @@ const triggerSet: TriggerSet<Data> = {
           // TODO: we could be fancier and say "Run through" or something for #2.
           en: 'Left Flank (or out)',
           de: 'Linke Flanke (oder raus gehen)',
+          fr: 'Suivez le flanc gauche (ou extérieur)',
```
Comment (VanSlanzar) [inline ui/raidboss/data/04-sb/trial/rathalos-ex.ts:73]: Here (normal and ex) I took into account your note and also the mechanics. The boss flicks his tail twice while turning twice, I find it more consistent to say to follow his flank.

File: ui/raidboss/data/04-sb/trial/tsukuyomi-ex.ts [hunk 2]
```diff
@@ -370,35 +369,40 @@ const triggerSet: TriggerSet<Data> = {
...
         'Concentrativity': 'Kenki concentré',
         'Crater': 'Explosion de fragment lunaire',
         'Dance Of The Dead': 'Danse des morts',
-        'Dark Blade': 'Lame ténébreuse',
         'Dispersivity': 'Onde Kenki',
         'Empire adds': 'Adds Impériaux',
         'Hagetsu': 'Pulvérisation lunaire',
         'Homeland adds': 'Adds Domiens',
-        'Lead Of The Underworld': 'Tir de l\'au-delà',
+        'Lead Of The Underworld(?!/Steel Of The Underworld)': 'Tir de l\'au-delà',
```
Comment (VanSlanzar) [inline ui/raidboss/data/04-sb/trial/tsukuyomi-ex.ts:387]: Here I don't know if there is a code to have a shorter line. If so tell me and I would edit.

File: ui/raidboss/data/04-sb/trial/tsukuyomi-ex.ts [hunk 3]
```diff
@@ -370,35 +369,40 @@ const triggerSet: TriggerSet<Data> = {
...
         'Lunar Rays': 'Rayons lunaires',
         'Midnight Rain': 'Bruine nocturne',
         'Moonbeam': 'Faisceau lunaire',
         'Moonfall': 'Impact de fragment lunaire',
         'Nightbloom': 'Lis araignée',
+        'Nightfall': 'Jeune nuit',
         'Perilune': 'Zénith lunaire',
         'Reprimand': 'Correction',
-        'Steel Of The Underworld': 'Pointes de l\'au-delà',
+        '(?<!Lead Of The Underworld/)Steel Of The Underworld': 'Pointes de l\'au-delà',
```
Comment (VanSlanzar) [inline ui/raidboss/data/04-sb/trial/tsukuyomi-ex.ts:399]: Same.

File: ui/raidboss/data/04-sb/trial/tsukuyomi-ex.ts (hunk 2 above)
Comment (MaikoTan) [reply ui/raidboss/data/04-sb/trial/tsukuyomi-ex.ts:387]: maybe just `(?!/)` for this usage.
'cause there is only this example following a slash.
also the bottom one.

File: ui/raidboss/data/04-sb/trial/rathalos-ex.ts (hunk 1 above)
Comment (quisquous) [reply ui/raidboss/data/04-sb/trial/rathalos-ex.ts:73]: Thanks for the note.  I think adding a "follow" verb here is fine, although the player will be instructed to follow twice.  In English, I thought it made the most sense to start with the most important word ("left").

File: ui/raidboss/data/04-sb/trial/tsukuyomi-ex.ts (hunk 2 above)
Comment (quisquous) [reply ui/raidboss/data/04-sb/trial/tsukuyomi-ex.ts:387]:
```suggestion
        'Lead Of The Underworld(?!/)': 'Tir de l\'au-delà',
```

> Here I don't know if there is a code to have a shorter line. If so tell me and I would edit.
Maiko is suggesting this.  Does this work?

File: ui/raidboss/data/04-sb/trial/tsukuyomi-ex.ts (hunk 3 above)
Comment (quisquous) [reply ui/raidboss/data/04-sb/trial/tsukuyomi-ex.ts:399]:
```suggestion
        '(?<!/)Steel Of The Underworld': 'Pointes de l\'au-delà',
```

Similarly, does this work?

**Comment by VanSlanzar** [comment]
Sorry, I had a big unexpected. Yes it works. Thank you @MaikoTan

**Pull request merged**